posthog/api/test/test_feature_flag.py

```diff
@@ -1332,3 +1332,23 @@ def assert_feature_flag_activity(self, flag_id: Optional[int], expected: List[Di
         self.assertEqual(
             activity, expected,
         )
+
+    def test_patch_api_as_form_data(self):
+        another_feature_flag = FeatureFlag.objects.create(
```

Comment:
it's a nit but we have a `self._create_flag_with_properties` method in the file to keep setup at the API level

Reply:
Actually that method has a weird surface. I won't pollute this change with that refactoring